Makefile

```diff
@@ -0,0 +1,25 @@
+BIN=node_modules/.bin
```

Comment:
We need a module bundler to build the UMD file and `webpack` does that, so it's ok.

Just change the `scripts` in `package.json` to something like this:
```json
// ...
"scripts": {
  "build": "npm run clean && npm run build:commonjs && npm run build:umd && npm run build:umd:min",
  "build:commonjs": "babel src --out-dir lib --ignore *-test.js",
  "build:umd": "NODE_ENV=development webpack",
  "build:umd:min": "NODE_ENV=production webpack",
  // ...
}
// ...
```